frontend/DESIGN_GUIDELINES.md

```diff
@@ -0,0 +1,2 @@
+# Design Guidelines for Job Triage
```

Comment:
Can we move this file somewhere common? Like `.github/contributing/`

Reply:
We can move this to the root folder. I have seen this in material-ui project where they have common documentation at root location.

Reply:
Yes, looks good. Can you make the changes?